docs/building_lbann.rst

```diff
@@ -62,10 +62,52 @@ Setup Spack (One-time setup)
           `Spack Documentation
           <https://spack.readthedocs.io/en/latest/configuration.html>`_.
 
+
+.. _install_lbann_as_user_known_center:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Building & Installing LBANN as a user at a known HPC Center
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With Spack setup and installed into your path, it can be used to
+install the LBANN executables. This approach is appropriate for users
+that want to train new or existing models using the Python front-end
+and are working at know HPC Computing Centers (e.g. LLNL LC, OLCF,
+ALCF, RIKEN).  The scripts that are invoked load known modules to tie
+into system install packages (e.g. MPI).
+
+1. Execute LBANN's user build script:
+
+   .. code-block: bash
+
+      /bin/bash -c "$(curl -fsSL https://github.com/LLNL/lbann/raw/develop/scripts/.build_lbann_as_user.sh)"
+
+   .. note::
+
+      There are three environment variables that can be used to
+      override the default:
+        - version of LBANN being installed (LBANN_VERSION),
+        - the variants enabled (LBANN_VARIANTS),
+        - and extra packages to build in the same enviroment (LBANN_VARIANTS).
+      The following is an example of Using the variables:
+
+      .. code-block: bash
+
+         LBANN_VERSION="develop" \
+         LBANN_VARIANTS="+cuda +numpy" \
+         LBANN_EXTRAS="-p py-torch@1.7.1 -p py-numpy" \
+         /bin/bash -c "$(curl -fsSL https://github.com/LLNL/lbann/raw/develop/scripts/.build_lbann_as_user.sh)"
+
+2. Once this is done, you can load LBANN:
+
+   .. code-block: bash
+
+      spack load lbann@<version>
+
 .. _install_lbann_as_user:
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Building & Installing LBANN as a user
+Building & Installing LBANN as a user (with maximum flexiblity)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I can't make a suggestion here for some reason, but the `~` need to go beyond the header text. Add more tildes to lines 109 and 111.